src/iSparrow/utils.py

```diff
@@ -8,14 +8,40 @@
 
 from pathlib import Path
 import validators as valid
-import importlib
+import importlib.util
+import os
+import time
+import yaml
 
 
 # custom exception to have some more control over what is raised
 class TFModelException(Exception):
     pass
 
 
+def read_yaml(path: str):
```

Comment:
Has been moved here from the set_up script, because it's needed in various places.